arch/arm/core/swap.S

```diff
@@ -324,6 +327,19 @@ _oops:
     blx _do_kernel_oops
     pop {pc}
 
+_escalate_priv:
```

Comment:
should we ifdef protect this?

Reply:
the pop {pc} is nefarious because it literally pops off the stack into the pc, and then voila, you are there.  But yeah I can block this off.